overlay/scripts/bootstrap.sh

```diff
@@ -139,6 +139,11 @@ if ! [ -z "${UPSTREAM_DNS}" ] ; then
   sed -i "s/#ENABLE_UPSTREAM_DNS#//;s/dns_ip/${UPSTREAM_DNS}/" /etc/bind/named.conf.options
 fi
 
+if ! [ -z "${DISABLE_DNSSEC_VALIDATION}" ] ; then
+  sed -i "s/dnssec_validation//;s/auto/no/" /etc/bind/named.conf.options
```

Comment:
Unless i'm mistaken, this sed line is wrong on several aspects
1) dnssec_validation should be dnssec-validation
2) The first half replaces dnssec-validation with nothing, then the second switches auto to no (anywhere in the config) so ```dnssec-validation auto;``` becomes ``` no;``` resulting in an invalid config if the typo was fixed